build.gradle

```diff
@@ -8,6 +8,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.6.20-RC'
```

Comment:
For the kotlin version, should we pick a released stable version?

Reply:
Done using 1.6.10